server/backend/quiz/views.py

```
@@ -77,6 +77,16 @@ def delete(self, request: Request) -> Response:
         deleted_count = Category.objects.filter(id__in=category_ids).delete()[0]
         return Response({"message": f"{deleted_count} categories and their associated data deleted."}, status=status.HTTP_200_OK)
         
+class UpdateQuizView(APIView):
+    permission_classes: ClassVar = [IsAuthenticated, IsAdminUser]
+
+    def put(self, request, category_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `put`

(ANN201)